docs/design/autoware-interfaces/ad-api/list/api/operation/mode/index.md

```diff
@@ -0,0 +1,40 @@
+# Operation Mode
+
+- {{ link_ad_api('/api/operation/mode/state') }}
+- {{ link_ad_api('/api/operation/mode/change_to_autonomous') }}
+- {{ link_ad_api('/api/operation/mode/change_to_stop') }}
+- {{ link_ad_api('/api/operation/mode/change_to_local') }}
+- {{ link_ad_api('/api/operation/mode/change_to_remote') }}
+- {{ link_ad_api('/api/operation/mode/engage') }}
+- {{ link_ad_api('/api/operation/mode/disengage') }}
+
+## Description
+
+As shown below, Autoware assumes that the vehicle interface has two modes, Autoware control and direct control.
+In direct control mode, the vehicle is operated using devices such as steering and pedals.
+If the vehicle does not support direct control mode, it is always treated as Autoware control mode.
+
+Autoware control mode has four operation modes. driver, local, remote, stop and autonomous.
```

Comment:
- Sort the order to be consistent with the enum order in the message.
- `driver` is unnecessary.
- Itemization looks good.
  - [IMO] Capitalization or using back quotes looks good to me.

```suggestion
Autoware control mode has four operation modes: `stop`, `autonomous`, `local`, and `remote`.
```

Or like this?

```suggestion
Autoware control mode has four operation modes.
1. Stop: Keep the vehicle stopped.
2. Autonomous: Autonomously control the vehicle.
3. Local: Manually control the vehicle from inside the vehicle by some device such as a joystick.
4. Remote: Manually control the vehicle from a web application on the cloud.
```